### ISSUE TITLE
Why not use Cross-validation technique?

### ISSUE THREAD

**Issue by moxiaokui** [WuJie1010/Facial-Expression-Recognition.Pytorch#84]
Hi, I want to know why don't you split the training set to training and validation set. It is normal to do that, but I don't see that in your code. Also, did the FER2013 winner used the cross-validation technique? Thanks.

**Comment by WuJie1010** [comment]
FER2013 has validation set itself, so we can train in the training set and validate in the validation set.
